testData/highlighting/commentStart.go

```diff
@@ -72,11 +75,11 @@ func Demo6() {}
 // Deprecated: use other thing
 func Demo7() {}
 
-// Demo8 does things  -> correct
+// Demo8 does things  -> incorrect
 //
 // Deprecated: use other thing
 
-func Demo8() {}
+func <weak_warning descr="'Demo8' should have a comment or not be exported">Demo8</weak_warning>() {}
```

Comment:
The answer to this is buried within the Go parser itself, I've yet to figure out where is the exact part that handles the documentation.